packages/in_app_purchase/in_app_purchase_platform_interface/lib/src/in_app_purchase_platform.dart

```diff
@@ -24,15 +24,16 @@ abstract class InAppPurchasePlatform extends PlatformInterface {
   /// The instance of [InAppPurchasePlatform] to use.
   ///
   /// Defaults to `null`.
```

Comment:
This is no longer correct. Instead, you should document that it must be set before using.

Reply:
done